test/command_line_test.exs

```diff
@@ -2,7 +2,7 @@ defmodule CommandLineTest do
   use ExUnit.Case
   import ExUnit.CaptureIO
 
-  test "displays message" do
+  test "(unit test) displays message" do
```

Comment:
That is a good point, I figured it might be good to have for current project consistency, but if we aren't expecting other types of tests here (or currently don't have any) I think you are right in the fact that this is unnecessary right now